src/test/java/org/cdc/gov/sdp/queue/DatabaseQueueComponentTest.java

```diff
@@ -70,7 +73,119 @@ public void testQueueConsumer() throws Exception {
 			assertEquals(1, rows_affected);
 
 			mock_endpoint.expectedMessageCount(rows_affected);
-			MockEndpoint.assertIsSatisfied(camelContext);
+			mock_endpoint.setResultWaitTime(5 * 1000);
+			mock_endpoint.assertIsNotSatisfied();
+
+			List<Map<String, Object>> lst = jdbcTemplate.queryForList(check_sent);
+			assertEquals(1, lst.size());
+		} finally {
+			jdbcTemplate.update(clear_dummy_data);
```

Comment:
Is there a race condition here?  Might we have a situation where the data is written after the clear?